app/views/articles/_search.html.erb

```diff
@@ -59,6 +59,16 @@
     }
   }
 
+  function addOptionalBanner(query, displayBanner, jobsUrl) {
+    if (query) {
+      var lowerCaseQuery = query.toLowerCase();
+
+      if (displayBanner && (lowerCaseQuery == "job" || lowerCaseQuery == "jobs")) {
```

Comment:
a cleaner alternative:
```suggestion
      if (displayBanner && lowerCaseQuery == ("job" || "jobs") {
```

do we want to handle the case when a user could search for "developer job" or "new job" or "tech jobs"? should we just do: `lowerCaseQuery.includes("job")`?

Reply:
I didnt want to include all instances of "job" initially bc I wasn't sure if we wanted to be showing the banner in the event someone searches "How to get a dev job" "How to do job interviews" in those cases, they are clearly seeking advice and I wasn't sure if throwing the banner up for them was too forward. What do you think?

Reply:
I don't feel like that banner is _too_ intrusive that a user would be annoyed if we tried to guess that they were looking for job-related content (maybe we'd get people looking at the jobs page who wouldn't ordinarily looked for it!)

...But I also see your point! I was mostly just asking to know if we specifically _didn't_ want to do that, which you have answered :) I'm ok with just basing the conditional off the single word, too.

Reply:
> looking at the jobs page who wouldn't ordinarily looked for it!)

Ya know, I kinda like this idea. People who love DEV but never considered they could work there I think it would be awesome to show to. I also think, especially when we have jr positions open we could get some people applying who might not have otherwise even thoughts about it. I'll let this sit for the weekend see if anyone else wants to chime in otherwise I will make your suggested change.

Reply:
woohoo 🎉 sounds great!

Reply:
+1 on adding it for all "jobs" related searches. maybe not "steve jobs" :D 
It'd be great if we could use a/b testing to see which strategy people prefer